src/main/java/uk/gov/pay/directdebit/payments/services/gocardless/GoCardlessEventToPaymentStateMapper.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.pay.directdebit.payments.services.gocardless;
+
+import uk.gov.pay.directdebit.common.model.DirectDebitStateWithDetails;
+import uk.gov.pay.directdebit.events.model.GoCardlessEvent;
+import uk.gov.pay.directdebit.payments.model.PaymentState;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static uk.gov.pay.directdebit.payments.model.PaymentState.SUCCESS;
+
+public class GoCardlessEventToPaymentStateMapper {
+    private static final Map<String, PaymentState> GOCARDLESS_ACTION_TO_PAYMENT_STATE = Map.of(
+            "failed", PaymentState.FAILED,
```

Comment:
Might be nice to have static import for `FAILED` since there's one for `SUCCESS`.

Reply:
👍 done